aqua/resources-api.aqua

```diff
@@ -28,29 +44,28 @@ func getResource(resource_id: ResourceId) -> ?Key, *Error:
   error: *Error
 
   resources: *Key
+  successful: *bool
   for n <- nodes par:
     on n:
       try:
         t <- Peer.timestamp_sec()
         get_result <- Registry.get_key_metadata(resource_id, t)
         if get_result.success:
           resources <<- get_result.key
+          successful <<- true
         else:
           error <<- get_result.error
 
-  timeout: ?string
-  join resources[0]
-  par timeout <- Peer.timeout(5000, "resource not found, timeout exceeded")
-
-  if timeout != nil:
-    error <<- timeout!
-
-  merge_result <- Registry.merge_keys(resources)
-
-  if merge_result.success:
-    result <<- merge_result.key
+  success <- wait(successful, 0)
```

Comment:
I'm kinda hesitant on hardcoding wait for only a single element...

I think it should be something like
```
const CONSISTENCY_LEVEL = 1

wait(successful, CONSISTENCY_LEVEL)
```